sw/airborne/modules/imu/imu.c

```diff
@@ -334,6 +334,91 @@ static void send_mag_current(struct transport_tx *trans, struct link_device *dev
 
 #endif /* PERIODIC_TELEMETRY */
 
+#if USE_SHELL
+#include "modules/core/shell.h"
+#include "printf.h"
+#include "string.h"
+
+static void show_calibrated(shell_stream_t *sh, struct imu_calib_t *calibrated) {
+  chprintf(sh, "  calibrated: neutral %d, scale %d, rotation %d, current %d, filter %d\r\n",
+      calibrated->neutral, calibrated->scale, calibrated->rotation, calibrated->current, calibrated->filter);
+}
+
+static void show_vect3(shell_stream_t *sh, char* name, struct Int32Vect3 *v) {
+  chprintf(sh, "  %s: %ld, %ld, %ld\r\n", name, v->x, v->y, v->z);
+}
+
+static void show_rates(shell_stream_t *sh, char* name, struct Int32Rates *r) {
+  chprintf(sh, "  %s: %ld, %ld, %ld\r\n", name, r->p, r->q, r->r);
+}
+
+static void show_matrix(shell_stream_t *sh, char* name, struct Int32RMat *m) {
+  chprintf(sh, "  %s:\r\n", name);
+  chprintf(sh, "    %ld, %ld, %ld\r\n", MAT33_ELMT(*m, 0, 0), MAT33_ELMT(*m, 0, 1), MAT33_ELMT(*m, 0, 2));
+  chprintf(sh, "    %ld, %ld, %ld\r\n", MAT33_ELMT(*m, 1, 0), MAT33_ELMT(*m, 1, 1), MAT33_ELMT(*m, 1, 2));
+  chprintf(sh, "    %ld, %ld, %ld\r\n", MAT33_ELMT(*m, 2, 0), MAT33_ELMT(*m, 2, 1), MAT33_ELMT(*m, 2, 2));
+}
+
+static void cmd_imu(shell_stream_t *sh, int argc, const char *const argv[])
+{
+  (void) argv;
+  int i = 0;
+  if (argc > 0) {
+    chprintf(sh, "Usage: rtos_mon\r\n");
```

Comment:
This says `rtos_mon` and should be `imu`

Reply:
copy/paste curse